DataFormats/Math/interface/GeantUnits.h

```diff
@@ -82,6 +82,12 @@ namespace geant_units {
       return (mm3 / 1.e9);
     }
 
+    template <class NumType>
+    inline constexpr NumType convertGeVToMeV(NumType gev)  // GeV -> MeV
```

Comment:
@casarsa thanks, I agree this is the cleanest solution. Not sure whether we have obvious immediate use cases, but for completeness I would add also the opposite conversion, i.e. MeV to GeV

Reply:
Done.